apps/lwipserver2/lwipserver2_imx8mm.camkes

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+import <std_connector.camkes>;
+import <global-connectors.camkes>;
+import <TimeServer/TimeServer.camkes>;
+import <SerialServer/SerialServer.camkes>;
+
+#include <camkes-lwip-ethernet-async.h>
+#include <camkes-lwip-base.h>
+#include <camkes-fdt-bind-driver.h>
+#include <camkes-single-threaded.h>
+#include <camkes-SerialServer-camkes-putchar-client.h>
+#include <camkes-BenchUtiliz.h>
+
+#include <echo/tuning_params.h>
+
+component FDT_device {
+    hardware;
+    emits FDT resource;
+}
+
+component LWIPServer {
```

Comment:
I wonder, is there a way to have one common generic CAmkES file with the server that includes the platform specific driver files? So this can be extended to other platforms easily without copy/pasting common things?